src/fixate/__init__.py

```diff
@@ -1 +1,11 @@
 __version__ = "0.5.7"
+
+import fixate.sequencer
+import fixate.config
+
+# Create the global sequencer here so it can be imported and
+#   used elsewhere with better traceability/type-hints/auto-complete
+global_sequencer = fixate.sequencer.Sequencer()
+# Plan to deprecate calls to RESOURCES[] and gradual replace.
+#   Keep here now for backwards-compatibility with test_scripts
+fixate.config.RESOURCES["SEQUENCER"] = global_sequencer
```

Comment:
I've been slowly trying to remove any object instantiation from happening at import time. Depending on how tangled things are, it might be a good change to do that here? The main entry point would create the Sequencer object and assign it to the global at startup. (this is probably something that should be deferred to a follow PR)

Reply:
Note that this could be a complete mess, because test scripts at the moment might rely on reference to the Sequencer at import time :(

Reply:
Reverted and moved to a separate branch. Nearly all our current test scripts fails the new atzip.pyz checks due to this, so will spend a bit more time cleaning it up there.